File: Beecrowd/Beginner/1000 - 1099/1020 - Age in Days/C++/1020_AgeinDays.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int ageInDays;
    cin >> ageInDays;

    int years = ageInDays / 365;
    ageInDays -= years * 365;

    int months = ageInDays / 30;
    ageInDays -= months * 30;

    cout << years << " ano(s)" << endl;
    cout << months << " mes(es)" << endl;
    cout << ageInDays << " dia(s)" << endl;
    
    return 0;
}

File: Beecrowd/Beginner/1000 - 1099/1020 - Age in Days/Python/1020_AgeinDays.py
age_in_days = int(input())

years = age_in_days // 365
age_in_days -= years * 365

months = age_in_days // 30
age_in_days -= months * 30

print(f"{years} ano(s)")
print(f"{months} mes(es)")
print(f"{age_in_days} dia(s)")